net/net.sh

```diff
@@ -176,8 +176,9 @@ else
 fi
 
 build() {
+  supported=("18.04")
   declare MAYBE_DOCKER=
-  if [[ $(uname) != Linux ]]; then
+  if [[ $(uname) != Linux || ! " ${supported[@]} " =~ " $(lsb_release -sr) " ]]; then
```

Comment:
@mvines, is this overkill? I was planning for a future when `supported=("18.04 18.10 19.04")`

Reply:
Na this is nice